cypress_test/integration_tests/desktop/writer/file_properties_spec.js

```diff
@@ -13,25 +13,21 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'File Property Tests', func
 
 	it('Add File Description.', function() {
 		writerHelper.openFileProperties();
+		// Too early click on the 'Description' button would have no effect.
+		cy.wait(500);
 		cy.cGet('#tabcontrol-2').click();
```

Comment:
I imagine its not as straightforward as the button is disabled until scrolled into place and async enabled just after cypress clicks on it.

Reply:
Hm, possible, I'll keep that in mind next time we run into something like this. I know click() loudly fails when you try to click on something not visible, but possibly it's different for the "visible, but disabled" case.